js/shopping-list-item.js

```diff
@@ -0,0 +1,40 @@
+class Item {
+  constructor(name, description) {
+    this.name = name;
+    this.description = description;
+    this.isDone = false;
+  }
+  check() {
```

Comment:
Again line breaks between your methods makes it easier to read the code by breaking up sections of the code into visible chunks.